src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -1114,7 +1114,7 @@ public bool NodeAutocompleteMachineLearningIsBeta
         {
             get
             {
-                return DynamoModel.FeatureFlags.CheckFeatureFlag("NodeAutocompleteMachineLearningIsBeta", false);
+                return DynamoModel.FeatureFlags?.CheckFeatureFlag("NodeAutocompleteMachineLearningIsBeta", false) ?? false;
```

Comment:
this causes an exception and makes it harder to debug

Reply:
@pinzart90 what do you think about forcing this member to be nullable? It seems like the API change is worth the wasted dev and user crash time.

Reply:
This API is only used in binding to a UI element. The binding is handled by a BooleanConverter which handles nullables the same way the function is written now.

Reply:
Let's leave it out of this PR.